provisioner/ibm-s3fs-provisioner_test.go

```diff
@@ -281,14 +281,36 @@ func getAutoDeletePersistentVolume() *v1.PersistentVolume {
 	}
 }
 
-func Test_Provision_BadPVCAnnotations(t *testing.T) {
+func Test_Provision_BadPVCAnnotations_AutoCreateBucket(t *testing.T) {
 	p := getProvisioner()
 	v := getVolumeOptions()
-	v.PVC.Annotations[annotationAutoCreateBucket] = "non-bool-value"
+	v.PVC.Annotations[annotationAutoCreateBucket] = "non-true-value"
 
 	_, err := p.Provision(v)
 	if assert.Error(t, err) {
-		assert.Contains(t, err.Error(), "cannot unmarshal PVC annotations")
+		assert.Contains(t, err.Error(), "invalid value for auto-create-bucket, expects true/false")
+	}
+}
+
+func Test_Provision_BadPVCAnnotations_AutoDeleteBucket(t *testing.T) {
+	p := getProvisioner()
+	v := getVolumeOptions()
+	v.PVC.Annotations[annotationAutoDeleteBucket] = "non-true-value"
+
+	_, err := p.Provision(v)
+	if assert.Error(t, err) {
+		assert.Contains(t, err.Error(), "invalid value for auto-delete-bucket, expects true/false")
+	}
+}
+
+func Test_Provision_Empty_SecretName(t *testing.T) {
+	p := getProvisioner()
+	v := getVolumeOptions()
+	v.PVC.Annotations[annotationSecretName] = ""
```

Comment:
Do we really need to specify annotation if its empty? Not specifying annotation wont be treated as empty?